roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -124,14 +129,18 @@ def local_images(self, images, task_vars):
         ]
 
     def is_image_local(self, image, task_vars):
-        result = self.module_executor("docker_image_facts", {"name": image}, task_vars)
+        """Check if image is already in local docker index."""
+        result = self.execute_module("docker_image_facts",
+                                     module_args={"name": image},
+                                     task_vars=task_vars)
```

Comment:
Since `module_args` is the second argument, we could omit its name.

```python
result = self.execute_module("docker_image_facts", {"name": image}, task_vars=task_vars)
```

Good catch on the wrong argument passing! :+1: :medal_military: 

I don't really know yet when `tmp` != `None`, but we should probably be forwarding the value of `tmp` that `run` receives as argument...